### DISCUSSION TITLE
about minemacs-ng

### DISCUSSION THREAD

**Discussion by donneyluck** [abougouffa/minemacs#173] [General]
I know that you have started the reconstruction of minemacs and can’t wait to experience the new version.
But I am a little worried because I have been using evil for many years and I am afraid it will be difficult to adapt to it.

**Comment by abougouffa** [comment]
Me too, I was using evil-mode for years, but at some point, I wanted to use Emacs as it is intended to be used. It takes some time to adapt to Emacs keybindings.

NB: If you switch to the minemacs-ng branch, you can still use Evil by adding `obsolete/me-evil` to the modules list.

**Reply by donneyluck** [reply]
Ok I will try the new version follow your lead

**Comment by donneyluck** [comment]
[nowisemacs](https://github.com/nowislewis/nowisemacs)
I recently looked at this configuration
borg setup    Use these two packages to manage config
meow  to manage edit mode
See if there is anything worth learning from
